web/src/src/pages/RecipeCreate/RecipeCreate.tsx

```diff
@@ -56,26 +54,11 @@ const LinkButton = styled.button`
     outline: none;
 `
 
-interface RecipePayload {
-    id: number,
-    name: string,
-    description: string,
-    ingredients: Array<RecipePayloadIngredient>
-}
-
-interface RecipePayloadIngredient {
-    name: string
-}
-
-interface Response {
-    data: RecipePayload
-}
-
 function RecipeCreate(req: any) {
     const id = req.match.params.id;
 
-    const [isEdit, setIsEdit] = useState(id !== null && id !== undefined)
-    const [recipeId, setRecipeId] = useState(id)
+    const [isEdit] = useState(id !== null && id !== undefined)
```

Comment:
it doesn't have sense to create a state you cannot update, it is just a constant

Reply:
this is still the same, maybe I am missing something, what's the reason to use a state there?